static-build/components/TestCard/styles.css

```diff
@@ -153,9 +153,13 @@
 
 /* visually hide bundler names */
 .iconList figcaption {
-  width: 0;
-  height: 0;
-  text-indent: -99999999px;
+  clip: rect(0 0 0 0);
```

Comment:
Can you use visiblity:hidden, display:none or 'width:0; height: 0; overflow: hidden'?

Reply:
I'm following guidance from this article https://css-tricks.com/inclusively-hidden/

Reply:
Thanks for the link. It's an interesting article. It seems that you want the element to be visible in some way, e.g. to assistive technology. Just curious, how does that work? Anyway, this seems an important use case, and might be a good candidate of css feature request. There must be something to improve if the simplest css property doesn't meet the requirement in the cases. Will talk to web platform people about this.